lib/mappers/in_mapper.py

```diff
@@ -0,0 +1,45 @@
+from akara import logger
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.qdc_mapper import QDCMapper
+
+class INMapper(QDCMapper):
+    def __init__(self, provider_data):
+        super(INMapper, self).__init__(provider_data)
+    
+    def map_alt_title(self):
+      self.source_resource_prop_to_prop("alternative")
+
+    def map_contributor(self):
+        self.source_resource_prop_to_prop("contributor")
+
+    def map_creator(self): 
+      self.source_resource_prop_to_prop("creator")
+
+    def map_created(self):
+      if exists(self.provider_data, "created"):
+            self.update_source_resource({"date": getprop(self.provider_data,
+                                                           "created")})
```

Comment:
The indentation of the original QDC mapping was not ideal, so I don't think it would be justifiable to break from it and indent things differently.  For example:
```
    def map_created(self):
        if exists(self.provider_data, "created"):
            self.update_source_resource({
                "date": getprop(self.provider_data, "created")
            })
```